src/config/oracle.php

```diff
@@ -16,6 +16,7 @@
         'edition'        => env('DB_EDITION', 'ora$base'),
         'server_version' => env('DB_SERVER_VERSION', '11g'),
         'load_balance'   => env('DB_LOAD_BALANCE', 'yes'),
+        'max_name_len'   => env('ORA_MAX_NAME_LEN', 30),
```

Comment:
Maybe we can use the `server_version` to identify the length like on PR https://github.com/yajra/laravel-oci8/pull/768?

`>=12` will set the length to 126 characters.